templates/zerver/hello.html

```diff
@@ -295,10 +284,28 @@
                     </p>
                 </div>
             </div>
+            <div class="col-3">
+                <div class="il-block">
```

Comment:
This seems to be in the wrong commit now.  This is moving text, and has nothing to do with creating a lighter scheme, correct?